mesh/node.c

```diff
@@ -1561,29 +1922,36 @@ static void get_managed_objects_cb(struct l_dbus_message *msg, void *user_data)
 			goto fail;
 
 		/* Generate device and primary network keys */
-		l_getrandom(dev_key, sizeof(dev_key));
-		l_getrandom(net_key.old_key, sizeof(net_key.old_key));
-		net_key.net_idx = PRIMARY_NET_IDX;
-		net_key.phase = KEY_REFRESH_PHASE_NONE;
-
-		if (!add_local_node(node, DEFAULT_NEW_UNICAST, false, false,
-						DEFAULT_IV_INDEX, dev_key,
-						PRIMARY_NET_IDX,
-						net_key.old_key))
+		l_getrandom(dev_key, 16);
```

Comment:
I would leave the `sizeof` and use that in other places too, instead of hardcoding `16`